src/content/guides/code-splitting.md

```diff
@@ -329,8 +330,8 @@ T> Using webpackPreload incorrectly can actually hurt performance, so be careful
 Once you start splitting your code, it can be useful to analyze the output to check where modules have ended up. The [official analyze tool](https://github.com/webpack/analyse) is a good place to start. There are some other community-supported options out there as well:
 
 - [webpack-chart](https://alexkuz.github.io/webpack-chart/): Interactive pie chart for webpack stats.
-- [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/): Visualize and analyze your bundles to see which modules are taking up space and which might be duplicates.
-- [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer): A plugin and CLI utility that represents bundle content as convenient interactive zoomable treemap.
+- [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/): Visualize and analyze your bundles to see which modules are taking up space and which are maybe duplicates.
```

Comment:
This change is not needed.

Reply:
@EugeneHlushko @montogeek @rishabh3112 What do you think?

Reply:
@EugeneHlushko @montogeek @rishabh3112 What do you think?

Reply:
i like `might be` better, in terms of how it reads

Reply:
@EugeneHlushko suggestion implemented